src/clients/patreon/patreonController.ts

```diff
@@ -1,29 +1,30 @@
 import { PATREON_URLS } from './constants';
-import { IPatreonController } from './patreonAgent';
-
-export interface PatreonUserTokens {
-  userAccessToken: string;
-  userRefreshToken: string;
-  userAccessTokenExpiry: Date;
-}
+import {
+  IPatreonController,
+  PatreonUserTokens,
+  PatronFullDetails,
+} from './patreonAgent';
+import uuid from 'uuid';
 
 export class PatreonController implements IPatreonController {
   private clientId = process.env.patreon_client_ID;
   private clientSecret = process.env.patreon_client_secret;
   private redirectUri = process.env.patreon_redirectURL;
 
-  public async getTokens(code: string): Promise<PatreonUserTokens> {
-    const getTokensUrl = new URL(PATREON_URLS.GET_TOKENS);
+  public async getPatreonUserTokens(code: string): Promise<PatreonUserTokens> {
+    const getPatreonUserTokensUrl = PATREON_URLS.GET_TOKENS;
```

Comment:
Let's create the new URL() object each time here because we're mutating it below. But let's move the links as a const string to the top of this file.